lunatrace/bsl/frontend/src/pages/project/dashboard/DefaultBranchSummary.tsx

```diff
@@ -0,0 +1,135 @@
+        </Card.Header>
+        <Card.Body>
+          <p>All new projects will be scanned soon after installation, check back in a few moments.</p>
+        </Card.Body>
+      </Card>
+    );
+  }
+  // const filteredFindings = filterFindingsByIgnored(build.findings);
+  // const vulnerablePackageCount = countCriticalVulnerabilities(filteredFindings);
+
+  const uploadDate = prettyDate(new Date(build.created_at as string));
+  const lastScannedDate = build.scans[0] ? prettyDate(new Date(build.scans[0].created_at as string)) : 'Never';
+
+  const filteredFindings = filterFindingsByIgnored(build.findings);
+  const vulnerablePackageCount = countCriticalVulnerabilities(filteredFindings);
+
+  const branch = branchName(build);
+  const branchUrl = branchLink(build);
+  const commitUrl = commitLink(build);
+
+  return (
+    <Card className="clickable-card w-100" onClick={(e) => navigate(`build/${build.id}`)}>
+      <Card.Header>
+        <Container fluid>
+          <Row>
+            <Col sm="6">
+              <Card.Title>
+                <h3>
+                  <span className="lighter">Latest Results</span>
+                  <ConditionallyRender if={build.git_branch}>
+                    <span className="darker"> on {build.git_branch}</span>
+                  </ConditionallyRender>
+                </h3>
+              </Card.Title>
+              <Card.Subtitle className="darker"></Card.Subtitle>
+            </Col>
+            <Col sm={{ span: 6 }}>
+              <div>
+                <Card.Title className="text-sm-end">
+                  <h3 style={{ display: 'inline' }}>{vulnerablePackageCount}</h3>
+                  <span className="darker"> critical packages</span>
+                </Card.Title>
+              </div>
+            </Col>
+          </Row>
+        </Container>
+      </Card.Header>
+      <Card.Body className="d-flex">
+        <Container fluid>
+          <Row>
+            <Col xs="12" sm={{ order: 'last', span: 5, offset: 4 }}>
+              <h6 className="text-sm-end">
+                <SourceIcon source_type={build.source_type} className="mb-1 me-1 lighter" />{' '}
+                <span className="darker"> From:</span> {uploadDate}
+              </h6>
+              <ConditionallyRender if={lastScannedDate !== uploadDate}>
+                <h6 style={{ textAlign: 'right' }}>
+                  <span className="darker"> Last scanned:</span> {lastScannedDate}
+                </h6>
+              </ConditionallyRender>
+            </Col>
+            <Col xs="12" sm="3">
+              <div className="build-git-info">
+                <ConditionallyRender if={branchUrl}>
+                  <h6>
+                    <span className="darker">Branch: </span>{' '}
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href={branchUrl || ''}
+                      onClick={(e) => e.stopPropagation()}
```

Comment:
Why do you need to stop the propagation here?

Reply:
because the parent element is also clickable